XRTK.Oculus/Packages/com.xrtk.oculus/Runtime/Providers/InputSystem/Controllers/OculusControllerDataProvider.cs

```diff
@@ -13,7 +13,7 @@
 using XRTK.Providers.Controllers;
 using XRTK.Services;
 
-namespace XRTK.Oculus.Providers.Controllers
+namespace XRTK.Oculus.Providers.InputSystem.Controllers
```

Comment:
Controllers aren't part of the input system. Even in core they're in their own folder:
![image](https://user-images.githubusercontent.com/13334553/80811494-0267d000-8b94-11ea-9e44-c1e13093d48f.png)